python/tank/pipelineconfig_utils.py

```diff
@@ -164,39 +174,182 @@ def get_core_path_for_config(pipeline_config_path):
     :param pipeline_config_path: path to a pipeline configuration
     :returns: Path to the studio location root or pipeline configuration root or None if not resolved
     """
+    try:
+        return _get_core_path_for_config(pipeline_config_path)
+    except:
+        return None
+
+def _get_core_path_for_config(pipeline_config_path):
+    """
+    Returns the core api install location associated with the given pipeline configuration.
+    In the case of a localized PC, it just returns the given path.
+    Otherwise, it resolves the location via the core_xxxx.cfg files.
+
+    :param str pipeline_config_path: path to a pipeline configuration
 
+    :returns: Path to the studio location root or pipeline configuration root or None if not resolved
+    :rtype: str
+
+    :raises TankFileDoesNotExistError: Raised if the core_xxxx.cfg file is missing for the
+        pipeline configuration.
+    :raises TankNotPipelineConfigurationError: Raised if the pipeline configuration path is not
```

Comment:
*Raised if the pipeline configuration path is not a pipeline configuration* - 😄  - eh? brexit means brexit?